js/sync-meta-box/index.js

```diff
@@ -106,7 +106,8 @@ const syncMetaBox = ($, isNewPost) => {
                 }
  
             } else {
-                check();
+                syncStatus.addClass('dlsc--wp-not-saved');
+                syncStatus.html('Draft not saved');
```

Comment:
This is the section that caused the infinite loop. I took the decision to revert back to the old text. This is showed when Wordpress and Draft are **not** in sync.